src/ansys/materials/manager/util/matml/matml_property_map.py

```diff
@@ -39,27 +39,51 @@
         ],
         "mappings": {},
     },
-    "Coefficient of Thermal Expansion::Isotropic": {
+    "Coefficient of Thermal Expansion::Isotropic::Secant": {
```

Comment:
Is there no default version here? Or does the Isotropic model always specify a subtype?